spec/services/push_notifications/send_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.describe PushNotifications::Send, type: :service do
+  let(:user) { create(:user) }
+  let(:params) do
+    {
+      user: user,
+      title: "Alert",
+      body: "some alert here",
+      payload: ""
+    }
+  end
+
+  context "with feature flag set to false/off" do
+    before { allow(FeatureFlag).to receive(:enabled?).with(:mobile_notifications).and_return(false) }
+
+    it "does nothing if the feature flag is disabled" do
+      expect do
+        described_class.call(params)
+      end.not_to change { Rpush::Client::Redis::Notification.all.count }
+    end
+  end
+
+  context "with no devices for user" do
+    before { allow(FeatureFlag).to receive(:enabled?).with(:mobile_notifications).and_return(true) }
+
+    it "does nothing" do
```

Comment:
```suggestion
    it "does nothing", :aggregate_failures do
```

But: is the first assertion (`expect(user.devices.count).to eq(0)`) really necessary. We care about the fact that the count doesn't change, the initial count does not seem relevant for that.